src/App.scss

```diff
@@ -0,0 +1,37 @@
+$whiteTwo: #e8e8e8;
+$background: #fafafa;
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {

```

Comment:
- [ ] The page should look the same as the Zeplin design, please add everything in the design e.g. `progress circle`, `profile button in the nav bar`, chapter`...

|Zeplin design|Your design|
|-|-|
|![Screenshot_20220609_052237](https://user-images.githubusercontent.com/25227022/172763569-0c3a3651-7693-4d89-b1aa-3138fbe97c4e.png)|![Screenshot_20220609_051908](https://user-images.githubusercontent.com/25227022/172763517-92155548-1324-4072-a64d-855d4e324ff6.png)|